gridcapa-task-manager-app/src/main/java/com/farao_community/farao/gridcapa/task_manager/app/FileManager.java

```diff
@@ -217,4 +218,18 @@ private String getFileNameFromUrl(String stringUrl) {
     public String generatePresignedUrl(String minioUrl) {
         return minioAdapter.generatePreSignedUrlFromFullMinioPath(minioUrl, 1);
     }
+
+    public void uploadFileToMinio(OffsetDateTime timestamp, MultipartFile file, String fileType, String fileName) {
+        String processTag = taskManagerConfigurationProperties.getProcess().getTag();
+        String path = String.format("MANUAL_UPLOAD/%s/%s", timestamp.format(ZIP_DATE_TIME_FORMATTER), fileName);
```

Comment:
Is it better to use constant for strings like "MANUAL_UPLOAD/%s/%s" ? 
Maybe its easier to read and prevent typo